certified-connectors/timeghost/apiDefinition.swagger.json

```diff
@@ -443,7 +443,7 @@
             "x-ms-summary": "Workspace"
           },
           {
-            "name": "$filter-Date range",
+            "name": "$filter-date-range",

```

Comment:
@MarcHochleutner-Koellisch It should be `"name": "$filter-Date range"` as per the submission you made in certification portal. If you are submitting a fix for an issue, please submit the artifacts in certification portal to get this fix applied to the certified connector and deploy to all the production regions

Reply:
Also, please confirm that this fix would not be affect to the existing users of your connector.

Reply:
Okay thanks, we will do that, @sriyen-msft. Yes, I confirm.